templates/firewall.unit.j2

```diff
@@ -6,6 +6,9 @@ After=syslog.target network.target
 Type=oneshot
 ExecStart=/etc/firewall.bash
 ExecStop=/sbin/iptables -F
+{% if firewall_enable_ipv6 %}
+ExecStop=/sbin/ip6tables -F
```

Comment:
Can you have multiple `ExecStop`s defined? According to my reading of [command lines](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Command%20lines) this _seems_ like it might work (e.g. it would run the first and second ExecStop commands successively), but it just feels weird. If you can confirm this works (both commands are run) I'm happy to merge. Thanks!